html/includes/graphs/sensor/temperature.inc.php

```diff
@@ -16,7 +16,7 @@
 $rrd_options .= ' AREA:sensor_min';
 $rrd_options .= ' AREA:sensor_diff#c5c5c5::STACK';
 
-$rrd_options .= " LINE1.5:sensor#cc0000:'".rrdtool_escape($sensor['sensor_descr'], 21)."'";
+$rrd_options .= " LINE1.5:sensor#cc0000:'".rrdtool_escape(ipmiSensorName($device['hardware'], $sensor['sensor_descr'], $ipmiSensorsNames), 21)."'"
```

Comment:
ipmiSensorName() is being called as a function here rather than part of an array - same for the other files.

This will also run all temp sensors through this regardless of it being an ipmi based sensor or not.